### ISSUE TITLE
Long pressing volume key

### ISSUE THREAD

**Issue by dotlambda** [SirCmpwn/sway#437]
My config contains
```sh
bindsym XF86AudioRaiseVolume exec --no-startup-id amixer -q set Master 3%+ unmute && killall -SIGUSR1 i3status
```
In i3, if I hold the key pressed, the command is executed multiple times. This is nice for decreasing/increasing volume.
Is this possible in sway?

**Comment by crondog** [comment]
I did a similar test with `bindsym XF86AudioRaiseVolume exec $term` and if I hold down XF86AudioRaiseVolume, heaps of terminals spawn. Is amixer working correctly?

**Comment by dotlambda** [comment]
It can't be amixer's fault: a simple `exec echo "foo" >> ~/bar` won't produce multiple lines when the key is held down, whereas in i3 this command does.
Is it maybe because the command doesn't start a graphical application?

**Comment by mikkeloscar** [comment]
I can reproduce this, and I think it's related to how wlc handles the return value of the `handle_key` callback: https://github.com/SirCmpwn/sway/blob/master/sway/handlers.c#L392

**Comment by mikkeloscar** [comment]
This has been fixed upstream: Cloudef/wlc@adbf9f09

**Comment by SirCmpwn** [comment]
Thanks @mikkeloscar!
